notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -55,10 +55,10 @@
    "source": [
```

Comment:
<p>need to add note that dGPU currently does not support INT4</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think it should be stated differently, like "There is no speedup for INT4 compressed models on dGPU". </p>

Reply:
<p>Done</p>

Reply:
<p>The same story for INT8 on dGPU.</p>

Reply:
INT8 for dGPU gives speedup but not significantly as I know, but ok

Reply:
Possibly it is better to rephrase there is no speedup -> there may not be speedup

Reply:
Rephrased